lib/pdf-editor-view.js

```diff
@@ -29,6 +29,7 @@ export default class PdfEditorView extends ScrollView {
     this.currentScale = 1.5;
     this.defaultScale = 1.5;
     this.scaleFactor = 10.0;
+    this.fitToWidth = atom.config.get('pdf-view.fitToWidth')
```

Comment:
Actually, I've gone ahead and made those changes in 0963a60bebbc12b13ea768a62ab7d7785e3512f9